src/lib/actions.js

```diff
@@ -0,0 +1,116 @@
+import {store} from "./store";
+import {saveUserUrl} from "./fb";
```

Comment:
Looks like there's a circular dependency where actions imports fb and fb imports actions.

Reply:
maybe it doesn't actually break anything tho...

Reply:
Yeah, there's nothing inherently wrong with circular deps in module-land, but it can create ambiguities during loading or rollup.

I'll have a think.

Reply:
I've resolved this by adding another var to the store telling us whether we _should_ request results.